qstnrs/questionnaire/migrations/0004_result_desc_result_upper_limit.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        """One Questionnaire can have one or more results.
+        We're moving the result description and upper limit from the Result
+        table into the Questionnaire table in order for one Questionnaire to
+        only have one possible result. Since currently the relationship between
+        Questionnaire and Result is one to many, we're only going to get data
+        from the first result record for a given questionnaire."""
+        for questionnaire in orm.Questionnaire.objects.all():
+            result_list = questionnaire.result_set.all()
+            if len(result_list) > 0:
+                questionnaire.result_description = result_list[0].description
+                questionnaire.result_upper_limit = result_list[0].upper_limit
```

Comment:
I don't think you need to store the upper limit. Upper limit is considered to be the max score one can have by selecting all the right answers, right?